sd-card/html/edit_reference.html

```diff
@@ -46,9 +46,9 @@
         }
     </style>
 
-    <link href="firework.css?v=$COMMIT_HASH" rel="stylesheet">
-    <script type="text/javascript" src="jquery-3.6.0.min.js?v=$COMMIT_HASH"></script>  
-    <script type="text/javascript" src="firework.js?v=$COMMIT_HASH"></script>
+    <link href="firework.css?v=019069c" rel="stylesheet">
```

Comment:
This is an unwanted change! `COMMIT_HASH` gets replaced when we build the firmware. We use this to make sure the Web UI always uses the latest files after an OTA Update (instead of the cached ones as normally)